analyzers/ntuple.py

```diff
@@ -74,10 +79,35 @@ def bookJet( tree, pName ):
     bookP4(tree, pName )
     for pdgid in pdgids:
         bookComponent(tree, '{pName}_{pdgid:d}'.format(pName=pName, pdgid=pdgid))
-    # var(tree, '{pName}_npart'.format(pName=pName))
+    var(tree, '{pName}_TCHP'.format(pName=pName))
+    var(tree, '{pName}_TCHP_IP'.format(pName=pName))
+    for tagger in ['IP_b_LL', 'IPs_b_LL', 'TCHE', 'TCHE_IP', 'TCHE_x', 'TCHE_y', 'TCHE_z', 'TCHE_xy', 'TCHE_pt', 'TCHE_dr']:
+        var(tree, '{pName}_{tagger}'.format(pName=pName, tagger=tagger))
+    for ptc in ['K0s', 'Kp', 'L0', 'S0', 'Sp', 'Sm', 'Muons'] :
+        var(tree, '{pName}_{ptc}'.format(pName=pName, ptc=ptc))
 
 def fillJet( tree, pName, jet ):
     fillP4(tree, pName, jet )
+    for tagger in ['IP_b_LL', 'IPs_b_LL', 'TCHE', 'TCHE_IP', 'TCHE_x', 'TCHE_y', 'TCHE_z', 'TCHE_xy', 'TCHE_pt', 'TCHE_dr']:
+        if jet.tags.get(tagger, None) is not None:
```

Comment:
if jet.tags contains the tagger, then you will fill in the value. 
You seem to think that it's a problem to have this value set to 0. why? when does this happen? 
if it does not, jet.tags.get returns None and None is False in a if. 

Can you try my suggestion and tell me what happens?